Checking the Genuineness of your Skywallet.md

```diff
@@ -0,0 +1,89 @@
+# Checking the Genuineness of your Skywallet
+
+## Checking your Skywallet Package for Tampering
+
+Once you receive the Skywallet shipment, please check for any physical tampering or damage of any sorts to the package.
+
+Tampering or damage can be anything from a small tear on the package to a package that is torn open.
+
+**Do not accept a shipment which seems to be damaged in any such way.**
+
+### An Untampered Shipment:
+
+<A Picture of an untampered shipment before opening>
+
+### A Tampered Shipment:
+
+<Optional picture of a tampered shipment which is not absolutely torn open but a small tear or hole so that the user will understand even such a small tamper can also spoil the device> <it can also be a general picture rather than a skywallet shipment>
+
+After opening the package, please ensure that the holographic sticker with the Skycoin logo on the Skywallet box is not broken.
+<A Picture of the unbroken holographic seal on the box>
+
+## Checking the Firmware
+
+The Skywallet is built to provide maximum security to the user.
+
+However, it is important to check whether your Skywallet is not compromised before using it.
+
+## Step 1 - Connecting Your Skywallet
+
+As the first step, you have to connect the Skywallet to your USB port.
+
+
+## Step 2 - Checking the message on your Skywallet's Display
+
+The display screen equipped on your Skywallet displays one of the following three messages when connected via USB port, marking whether your Skywallet is authentic.
+
+### Scenario 1
+
+If the Skywallet is a genuine one and is being used for the first time, then the screen will display the following message:
+
+**Welcome!**
+
+**Please visit skycoin.net** 
+
+Along with the Skycoin Logo.
+
+<A photo of the Hardware wallet displaying the welcome message>
+
+A Skywallet that is being used for the first time will not have a firmware(the software interface) installed. 
+
+Hence any message apart from this welcome message implies that your Skywallet has been tampered by a third party.
```

Comment:
has been tampered with by

Reply:
Also add here that any other message means  they should not use  the wallet under any circumstance.